ENIGMAsystem/SHELL/Platforms/General/POSIX/main.cpp

```diff
@@ -3,6 +3,8 @@
 #include <limits.h>
 #include <unistd.h>
 #include <libgen.h>
+#include <string.h>
+#include <stdio.h>
```

Comment:
Same for `cstdio`.

Reply:
Oh, right, the C++ headers are preferred. My bad.